x/fungible/types/message_remove_foreign_coin.go

```diff
@@ -10,10 +10,10 @@ const TypeMsgRemoveForeignCoin = "remove_foreign_coin"
 
 var _ sdk.Msg = &MsgRemoveForeignCoin{}
 
-func NewMsgRemoveForeignCoin(creator string, name string) *MsgRemoveForeignCoin {
+func NewMsgRemoveForeignCoin(creator string, zrc20_address string) *MsgRemoveForeignCoin {
```

Comment:
```suggestion
func NewMsgRemoveForeignCoin(creator string, zrc20Address string) *MsgRemoveForeignCoin {
```
Variable names use camel case in Go